src/peerconnection.cc

```diff
@@ -64,7 +65,20 @@ PeerConnection::PeerConnection(RTCConfiguration configuration)
   _factory = PeerConnectionFactory::GetOrCreateDefault();
   _shouldReleaseFactory = true;
 
-  _jinglePeerConnection = _factory->factory()->CreatePeerConnection(configuration, nullptr, nullptr, nullptr, this);
+  auto portAllocator = std::unique_ptr<cricket::PortAllocator>(new cricket::BasicPortAllocator(
+              _factory->getNetworkManager(),
+              _factory->getSocketFactory()
+          ));
+  portAllocator->SetPortRange(
+      configuration.portRange.min.FromMaybe(0),
+      configuration.portRange.max.FromMaybe(65535));
```

Comment:
bracket on separate line ?

Reply:
Ah, good catch. I moved the bracket to the same line instead.